nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -174,6 +253,251 @@ in
         '';
       };
 
+      desync = lib.mkOption {
+        default = { };
+        description = ''
+          A set of nixpkgs copies that can be updated independently of the main nixpkgs.
+
+          It allows the system to keep building even if the latest version of the channel
+          has a broken build for a package.
+
+          If it is used, then the `nixos-upgrade-desyncs` command is added to the system
+          environment, that can be used to try to upgrade the desyncs to the current channel
+          version.
+          If they build fine then they will be upgraded, if not they will be left at their
+          current setting.
+
+          Using `maxDesyncAge`, you can limit the number of rebuilds that one nixpkgs
```

Comment:
If `maxDesyncAge` is an option, could you link to it?

Reply:
You'd need to use the relevant "anchor" tag from the [nixos manual's options appendix](https://nixos.org/manual/nixos/unstable/options).

It should be `#opt-system.autoUpgrade.desync.maxDesyncAge`, but to be sure you can build the manual locally using `nix-shell ./nixos/doc --run devmode`.

The syntax will look like ``[`maxDesyncAge`](#opt-system.autoUpgrade.desync.maxDesyncAge)`` or `[](#opt-system.autoUpgrade.desync.maxDesyncAge)` to automatically use the full option name as the link text.

Reply:
Oh yes now that you say it that does ring a bell, thank you! I'm surprised that there aren't more options making use of that, if there were enough examples to copy-paste from it'd make things much easier… well at least after this PR the auto-upgrade module will be such an example!